dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcCompletenessTableManager.java

```diff
@@ -261,10 +261,10 @@ private List<AnalyticsTableColumn> getColumns() {
         "concat(ds.uid,'-',ps.iso,'-',ous.organisationunituid,'-',acs.categoryoptioncombouid) as id ";
     String diffInSeconds = sqlBuilder.differenceInSeconds("cdr.date", "ps.enddate");
     String timelyDateDiff = diffInSeconds + " / (" + SECONDS_PER_DAY + ")";
+
     String timelyAlias = "((" + timelyDateDiff + ") <= ds.timelydays) as timely";
 
-    List<AnalyticsTableColumn> columns = new ArrayList<>();
-    columns.addAll(FIXED_COLS);
+    List<AnalyticsTableColumn> columns = new ArrayList<>(FIXED_COLS);
```

Comment:
I know this is opiniated, but we have it like this for all table manager classes, so good to keep it consistent.